modules/045-snapshot-controller/templates/snapshot-controller/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: d8-snapshot-controller
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-controller" "workload-resource-policy.deckhouse.io" "master")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: d8-snapshot-controller
+  updatePolicy:
+    updateMode: "Auto"
+{{- end }}
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  annotations:
+    helm.sh/hook: post-upgrade, post-install
+    helm.sh/hook-delete-policy: before-hook-creation
+  name: d8-snapshot-controller
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-controller" )) | nindent 2 }}
+spec:
+  minAvailable: {{ include "helm_lib_is_ha_to_value" (list . 1 0) }}
+  selector:
+    matchLabels:
+      app: d8-snapshot-controller
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Can't see anything related to monitoring. Does the controller and webhook export any metrics at all?